SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -681,16 +681,17 @@ ASpatialFunctionalTestFlowController* ASpatialFunctionalTest::GetFlowController(
 			return FlowController;
 		}
 	}
+	verifyf(false, TEXT("We should always find a flow controller."));
 	return nullptr;
 }
 
 APlayerController* ASpatialFunctionalTest::GetFlowPlayerController(const ESpatialFunctionalTestWorkerType WorkerType, const int WorkerId)
 {
-	if (ASpatialFunctionalTestFlowController* FlowController = GetFlowController(WorkerType, WorkerId))
-	{
-		return Cast<APlayerController>(FlowController->GetOwner());
-	}
-	return nullptr;
+	ASpatialFunctionalTestFlowController* FlowController = GetFlowController(WorkerType, WorkerId);
+	APlayerController* PlayerController = Cast<APlayerController>(FlowController->GetOwner());
+	verifyf(IsValid(PlayerController), TEXT("The parent of a FlowController should always be a valid PlayerController."));
```

Comment:
these are actually equivalent to checks here

Reply:
changed to checks!